src/wazuh_db/wdb_agent.c

```diff
@@ -103,15 +110,33 @@ int wdb_update_agent_version(int id, const char *os_name, const char *os_version
     char wdbquery[OS_BUFFER_SIZE] = "";
     char wdboutput[OS_BUFFER_SIZE] = "";
     int wdb_sock = -1;
-  
+ 
+    // os_uname fails with %Q flag
```

Comment:
Just by curious... why? 😁

Reply:
The bug generated a `segmentation fault` in `sqlite3_snprintf()`.
It was not clear to me if it was related to the length of the parameter, a character inside it, or something else.